app/controllers/appointments_controller.rb

```diff
@@ -9,9 +9,11 @@ def new
   def create
     @shop = Shop.find(params[:appointment][:shop_id])
     @appointment = Appointment.new(appointment_params.merge(user_id: @user.id,
-                                                             shop_id: @shop.id))
+                                                            shop_id: @shop.id))
     if @appointment.save
       flash[:success] = 'The appointment has been added'
+      AppointmentMailer.user_email(@appointment).deliver_now
```

Comment:
use a better name for the email. one that suggests what the email is for. Don't forget to rename the template

Reply:
user_email has been changed to user_appointment_confirmation_email